packages/react-components/src/index.ts

```diff
@@ -55,7 +73,42 @@ export type {
   StrokeWidthTokens,
   Theme,
 } from '@fluentui/react-theme';
-export { SSRProvider } from '@fluentui/react-utilities';
+export {
+  clamp,
+  getNativeElementProps,
+  getNativeProps,
+  getPartitionedNativeProps,
+  getRTLSafeKey,
+  getSlots,
+  omit,
+  resolveShorthand,
+  shouldPreventDefaultOnKeyDown,
+  useBoolean,
```

Comment:
`useBoolean` `useConst` are simply different riffs of refs and state, which aren't very complicated by themselves. I personally don't think it's that useful for apps to be able to use them from Fluent.

`const [boolean, setBoolean] = React.useState(false)` is effectively `useBoolean`

`const constRef = React.useRef(valueThatNeverUpdates)` or `const constRef = React.useState(() => valueThatNeverUpdates)` is basically `useConst`

IMO I would remove these even internally since they are so simple to use with native hooks